src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1253,6 +1255,16 @@ public static class ColorGrouping
             public static string ColorGroupingDialogDescription = "Select metadata as criteria for color grouping";
             public static string ApplyColorGroupingButton = "Apply Color Grouping";
          }
+
+         public static class MultiCurveOptions
+         {
+            public static string CurrentValue = "<Current value>";
+            public static string Color = "Color";
```

Comment:
We have those constants somewhere. Please reuse. They clearly mean the same

Reply:
done